<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExceptionPopup</class>
 <widget class="QDialog" name="ExceptionPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unhandled Exception</string>
  </property>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>105</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textException">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>821</width>
     <height>251</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
